Dockerfile

```diff
@@ -0,0 +1,44 @@
+#FROM ocm-container
+FROM fedora:latest
```

Comment:
This should probably be a tagged image version that we consciously update when a new version comes out.  I've previously had bad experiences when "latest" changed to point to a newer version and some things broke.  We should ensure that the Dockerfile is in a working tested state, and we can't do that when "latest" can arbitrarily point to a new version.

Reply:
I have mixed feelings about it.  On one hand, we have issues if they update the upstream.  On the other hand, we have to have someone watch the upstream and update our build when necessary.

For a project like this, I think it's safe to stay on latest but be cognizant of that, especially once we get to the point where you won't be building this locally unless you're heavily customizing it or if you're building the release, in which case we'd probably want the latest updated at that point anyway.

With the end goal of providing a quay.io image that the average user of ocm-container can just download and run, as long as we tag _those_ appropriately I'm in the camp of leaving this at :latest, but if I'm the minority I'll concede the argument.

Reply:
I would argue you have to watch something anyway.  You already have versions of the binaries hard-coded, so at the point they need to be upgraded, a human has to increment a number, just like this.

If you were to keep `latest` and switch to using tests to verify, then you need a human watching the build tests, unless PRs fail to build.  I would say as long as each PR is built as part of it's testing, then that is probably the best solution:

* leave as latest
* move other binaries to latest
* make each PR test require an image build

Reply:
There is a consideration moving to automation for image builds and tests. This should alert us to problems as they happen.